index.html

```diff
@@ -50,6 +58,190 @@ <h1>Hey there.
+                    <li class="taging"><p class="tag">bootstrap</p></li>
+                    <li class="taging"><p class="tag">Ruby</p></li>
+                </ul>
+                <button type="button">See Project</button> 
+            </div>
+
+
+            <section class="frame">
+
+                <div class="cards">
+                    <h2 class="cardtitle">Profesional Art Printing Data</h2>
+                    <p class="cardsupporting">
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required. 
+                        has been the industry's standard
+                    </p>
+                    <ul class="tags catags">
+                        <li class="taging cataging"><p class="tag catag">html</p></li>
+                        <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                        <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                    </ul>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div>
+
+                <div class="cards">
+                    <h2 class="cardtitle">Profesional Art Printing Data</h2>
+                    <p class="cardsupporting">
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required. 
+                        has been the industry's standard
+                    </p>
+                    <ul class="tags catags">
+                        <li class="taging cataging"><p class="tag catag">html</p></li>
+                        <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                        <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                    </ul>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div>
+
+                <div class="cards">
+                    <h2 class="cardtitle">Profesional Art Printing Data</h2>
+                    <p class="cardsupporting">
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required. 
+                        has been the industry's standard
+                    </p>
+                    <ul class="tags catags">
+                        <li class="taging cataging"><p class="tag catag">html</p></li>
+                        <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                        <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                    </ul>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
```

Comment:
- [ ] [Optional] You have implemented the `hover` effect but the `pressed` effect is not working. Kindly implement it
### Design
![ff](https://user-images.githubusercontent.com/81727988/177008691-547e343a-e3a3-4bc4-a6e2-0d7950d2edca.png)